app/assets/scss/_brief_submission.scss

```diff
@@ -1,3 +1,20 @@
+.instruction-list-item {
```

Comment:
Shouldn't this all get pushed back to the instruction list in the toolkit, rather than being in just this app?